lib/phlex/view.rb

```diff
@@ -27,12 +27,15 @@ def compiled?
 			end
 		end
 
+    attr_reader :helpers
```

Comment:
Although an `attr_reader` is faster, I think we should probably just define this as a method and delegate to the `@_view_context` instance variable, and then inline that in a compilation step.

```suggestion
    def helpers
      @_view_context
    end
```

Reply:
Do you want the method to be at the top of the public API?

Reply:
I think it's fine here. It's tricky to know where to document this as it's not a Rails-specific feature, but it is a view_context specific feature so it works in Bridgetown, etc. I think for now, this is fine here but should be documented under Rails.